README.md

```diff
@@ -91,14 +91,36 @@ for (auto const& result : results) {
 
 Prepare the CMake build directory:
 
-```
+```sh
 mkdir build-release
 cd build-release
 cmake ..
 ```
 
-Optionally, also prepare the debug environment:
+Starting with GCC 8.0, some warnings will arise, which will run into an error.
```

Comment:
We shouldn't need these since https://github.com/puppetlabs/libwhereami/pull/43 will take care of the warnings.

Reply:
fixed it